packages/suite-data/files/translations/en.json

```diff
@@ -1282,8 +1282,7 @@
     "TR_SUITE_WEB_LANDING_HOW_TO_VERIFY": "How to verify and run on Linux",
     "TR_SUITE_WEB_LANDING_LINUX_ARM64_LABEL": "for Linux (arm64)",
     "TR_SUITE_WEB_LANDING_LINUX_LABEL": "for Linux (x86_64)",
-    "TR_SUITE_WEB_LANDING_MACOS_ARM64_LABEL": "for macOS (Apple Silicon)",
-    "TR_SUITE_WEB_LANDING_MACOS_LABEL": "for macOS (Intel)",
```

Comment:
`for macOS (Apple Silicon) / for macOS (Intel)` are the correct versions for September release but `for MacOS` is what we want for the August release. Maybe we should start syncing translations after freeze on release branch directly...